src/bin/mission_manager/machine.h

```diff
@@ -1627,6 +1627,8 @@ struct PoweredDescent
     goby::time::SteadyClock::time_point powered_descent_timeout_;
     // determines when the bot is diving
     bool is_bot_diving_{false};
+    // initial depth reading
```

Comment:
Not sure if we need this comment given the name of the variable

Reply:
Changed comment to give more insight on what the variable is used for